styles/twitter/catppuccin.user.css

```diff
@@ -153,6 +151,13 @@
     // white text, seems to appear on accent colors
     .r-jwli3a {
       color: if(@lookup=latte, #fff, @crust);
+
+      // cw svg
+      &:has(
+          path[d="M3.693 21.707l-1.414-1.414 2.429-2.429c-2.479-2.421-3.606-5.376-3.658-5.513l-.131-.352.131-.352c.133-.353 3.331-8.648 10.937-8.648 2.062 0 3.989.621 5.737 1.85l2.556-2.557 1.414 1.414L3.693 21.707zm-.622-9.706c.356.797 1.354 2.794 3.051 4.449l2.417-2.418c-.361-.609-.553-1.306-.553-2.032 0-2.206 1.794-4 4-4 .727 0 1.424.192 2.033.554l2.263-2.264C14.953 5.434 13.512 5 11.986 5c-5.416 0-8.258 5.535-8.915 7.001zM11.986 10c-1.103 0-2 .897-2 2 0 .178.023.352.067.519l2.451-2.451c-.167-.044-.341-.067-.519-.067zm10.951 1.647l.131.352-.131.352c-.133.353-3.331 8.648-10.937 8.648-.709 0-1.367-.092-2-.223v-2.047c.624.169 1.288.27 2 .27 5.415 0 8.257-5.533 8.915-7-.252-.562-.829-1.724-1.746-2.941l1.438-1.438c1.53 1.971 2.268 3.862 2.33 4.027z"]
+        ) {
+        color: @text;
```

Comment:
Is there not a `fill="#000000"` or something you can select? How is the color being set on Twitter's end that you have to override it like this?

Reply:
nope! there isn't. and content warnings are a special case where it should be the text color. though i should probably fix it on latte but that's for another day...